wiki/Ranking_Criteria/en.md

```diff
@@ -31,22 +31,17 @@ Glossary
 
 ### Metadata
 
+- **Official sources**: Places directly from the artist or their label such as official websites, CD Scans, or official uploads to websites like Bandcamp, YouTube, or SoundCloud. Third party websites such as wikis, databases (vndb, vgmdb, etc.), or music services (Spotify, iTunes, Amazon, etc.) do not count as official sources.
 - **vs.**: versus
-- **&**: and
-- **feat./ft.**: featuring
+- **feat.**: featuring
 - **CV**: character voice, usually used to refer to the voice actor of a fictional character.
-- **Lenticular brackets**: Also known as Sumitsukikakko and written as `【】`.
-- **Corner brackets**: Also known as Kagikakko and written as `「」`.
-- **Square brackets**: `[]`
-- **asterisk**: `*`
-- **umlauts**: `ü`, `ö`, `ä` and `ß`
-- **TV Size**: A song where the particular version is used in a television program, web series, or direct to video series, such as an opening, ending, or insert song. Indie/doujin series are not included in this.
-
-Note: If a beatmapset's song contains matching sections to the TV size song, in the same order, and is roughly the same length as the official TV Size song, the beatmapset's edit will also count as TV Size
-
+- **Asterisk**: `*`
+- **Umlauts**: `ü`, `ö`, `ä` and `ß`
+- **Whitespace**: A visual spacing between characters, not always a literal space. If a character already has space in it, such as all of Japanese, that counts as whitespace.
+- **TV Size**: A song where the particular version is used in a television program, web series, or direct to video series, such as an opening, ending, or insert song. Fan-made works are not included in this.
```

Comment:
```suggestion
- **TV Size**: A cut of a song that is used in a television program, web series, or video series, such as an opening, ending, or insert song. Fan-made works are not included in this.
```
probably also mention that these are traditionally shorter versions of their full/original counterparts

Reply:
i dont think that's always the case - sometimes the version used on TV is the full song, a tv size does not have to be cut of a full song, it just needs to have been aired on tv in that version (though in most cases you are right)

Reply:
yeah that's what I meant by "traditionally"